### DISCUSSION TITLE
Update won't stop loading after clicking a stop(cancel) button

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15652] [Q&A] [answered]
https://user-images.githubusercontent.com/86934170/142014156-d6ae0517-1b5c-4c80-a326-545e13ee4f80.mp4

In this case, in my opinion, the button should stop the update/beatmap download process.
Then the scenario for the behavior of this button should be as follows: when an action is ongoing, it must cancel (stop) this action; with other notifications must hide it.

**Comment by peppy** [comment, marked as answer]
Probably shouldn't have a cancel button.
